go.mod

```diff
@@ -1,6 +1,8 @@
 module go.k6.io/k6
 
-go 1.20
+go 1.21
+
+toolchain go1.22.6
```

Comment:
This effectively also removes 1.21 from the versions that will be used. 

I am not against this as we in practice likely will upgrade to 1.23 for v0.54 (?) but still probably relevant to be added to the description.